spring-cloud-app-broker-docs/build.gradle

```diff
@@ -31,22 +25,17 @@ configurations {
 	docs
 }
 
-dependencyManagement {
-	imports {
-		mavenBom "org.springframework.boot:spring-boot-dependencies:${springBootVersion}"
-	}
-}
-
 dependencies {
+	implementation enforcedPlatform("org.springframework.boot:spring-boot-dependencies:${springBootVersion}")
 	implementation project(":spring-cloud-app-broker-core")
-	implementation("org.springframework.boot:spring-boot-starter")
-	implementation("org.springframework.boot:spring-boot-starter-security")
-	implementation("org.springframework.boot:spring-boot-starter-tomcat")
-	implementation("io.projectreactor:reactor-core")
-	implementation("org.springframework.boot:spring-boot-starter-data-r2dbc:2.3.0.RELEASE")
-	implementation("io.r2dbc:r2dbc-h2:0.8.0.RELEASE")
+	implementation "org.springframework.boot:spring-boot-starter"
+	implementation "org.springframework.boot:spring-boot-starter-security"
+	implementation "org.springframework.boot:spring-boot-starter-tomcat"
+	implementation "io.projectreactor:reactor-core"
+	implementation "org.springframework.data:spring-data-r2dbc:1.1.1.RELEASE"
```

Comment:
Can we not use the boot r2dbc starter from boot 2.3?

Reply:
We can. I hit something that caused me to change this, but now things have settled again, the starter works just fine.

Reply:
I just realized we haven't bumped the project to boot 2.3 yet, so I'm not sure it does make sense to mix that version in with the boot 2.2 dependencies. Probably best to leave it as is, and then update these dependencies when the project is upgraded.

Reply:
I have updated to boot 2.3 :)